core/src/main/resources/filodb-defaults.conf

```diff
@@ -448,7 +448,7 @@ filodb {
 
     circuit-breaker {
       # query circuit breaker enabled or not
-      enabled = false
+      enabled = true
```

Comment:
We don't have data to see enable this by default, let's not enable this feature.

Reply:
What data do we need? And why can't we collect this data in a perf test during this release ?